packages/common/src/logger.ts

```diff
@@ -43,6 +43,10 @@ import { ServiceTypes } from '../declarations'
 import config from './config'
 import { logsApiPath } from './schema.type.module'
 
+import { EditorState } from '@ir-engine/editor/src/services/EditorServices'
```

Comment:
this will cause import issues, the logger has a context state that can be injected by various routes `packages/client-core/src/util/ClientContextState.ts`